e2e/e2eutils.go

```diff
@@ -34,7 +34,10 @@ func SetupGitRepo(t *testing.T) (*testutil.TestGitRepo, string, func()) {
 		t.FailNow()
 	}
 
-	g, _, c := testutil.SetupDefaultRepoAndWorkspace(t, testutil.Dataset1, map[string]string{})
+	g, _, c := testutil.SetupDefaultRepoAndWorkspace(t, testutil.Content{
```

Comment:
Let's followup on the general scope of this test framework, there are plans for having e2e testing of functions, we need to rationalize the different appraoches.